### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'then')

### ISSUE THREAD

**Issue by pyroMain404** [johannesjo/super-productivity#3270]
### Steps to Reproduce

1. Start the Initial Tour in Web Mode
2. Keep going normally until the Sync section
3. Once the Sync section is done, this error pops up

I tried using Chrome, Edge and Vieb (an open source browser based on Electron and Chromium). It's consistent throughout these browsers.

### Meta Info
META: SP9.0.3 Browser – it – Win32 – Mozilla/5.0 (Window NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36

### Stacktrace
```
TypeError: Cannot read properties of undefined (reading 'then')
    at show (https://app.super-productivity.com/main-XULEE4CZ.js:186:5567)
    at o.when.show (https://app.super-productivity.com/main-XULEE4CZ.js:198:48125)
    at https://app.super-productivity.com/main-XULEE4CZ.js:198:2781
    at Array.forEach (<anonymous>)
    at iM.trigger (https://app.super-productivity.com/main-XULEE4CZ.js:198:2723)
    at iM._show (https://app.super-productivity.com/main-XULEE4CZ.js:198:40513)
    at iM.show (https://app.super-productivity.com/main-XULEE4CZ.js:198:38567)
    at j7.show (https://app.super-productivity.com/main-XULEE4CZ.js:198:45752)
    at j7.next (https://app.super-productivity.com/main-XULEE4CZ.js:198:45278)
    at e.next (https://app.super-productivity.com/main-XULEE4CZ.js:198:47765)
```

### Actions Before Error
```
1721391424362: [Task][Issue] Add Task 
1721391424366: [Persistence] Save to DB (2) 
1721391424366: [Task] Move tasks to other project 
1721391424368: [Persistence] Save to DB (2) 
1721391426286: [Layout] Hide AddTaskBar 
1721391430771: [Task] Toggle start 
1721391430772: [Persistence] Save to DB 
1721391430772: [Task] SetCurrentTask 
1721391431480: [Task] Add time spent 
1721391431481: [Project] Update Work End 
1721391431481: [Tag] Update Work End for Tag 
1721391431483: [Persistence] Save to DB (3) 
1721391431832: [Task] Toggle start 
1721391431834: [Persistence] Save to DB 
1721391431834: [Task] UnsetCurrentTask 
1721391433608: [Task] SetSelectedTask 
1721391433609: [Layout] Hide Notes 
1721391435568: [Task] SetSelectedTask 
1721391444799: [Task] Update Task 
1721391444801: [Persistence] Save to DB 
1721391447245: [Task] Update Task 
1721391447440: [Persistence] Save to DB 
1721391448329: [Task] Update Task 
1721391448330: [Persistence] Save to DB 
1721391450355: [Task] Delete Task 
1721391450358: [Persistence] Save to DB (3) 
1721391450359: [Task] UnsetCurrentTask 
1721391463816: [Global Config] Update Global Config Section 
1721391463833: [Persistence] Save to DB 
1721391463833: [Dropbox] Trigger Auth Dialog
```

**Comment by johannesjo** [comment]
Thank you very much! This will be fixed with the next release!

**Comment by johannesjo** [comment]
Also a duplicate of #3267 btw :)
